core/src/main/kotlin/com/exactpro/th2/read/db/bootstrap/Main.kt

```diff
@@ -187,19 +193,45 @@ internal fun setupApp(
     READINESS_MONITOR.enable()
 }
 
-private fun <BUILDER> createReader(
+private fun createMessageLoader(
+    factory: CommonFactory,
+    componentBookName: String
+) = runCatching {
+    MessageSearcher.create(factory.grpcRouter.getService(DataProviderService::class.java)).run {
+        object : MessageLoader {
+            override fun load(dataSourceId: DataSourceId, properties: Map<String, String>): TableRow? =
+                findLastOrNull(
+                    book = componentBookName,
+                    sessionGroup = dataSourceId.id,
```

Comment:
The moment that you forgot to set a session group in the request might be a sign that we should not use a default parameter in the `findLastOrNull` method at all. Just try to imagine that somebody uses the `MessageSearcher` and passes it to some other class to use. How do you know which session group and alias are requested?
Also, you can set message streams and don't set the session group (like you did). What is the behavior the user should expect from this combination? I believe we might provide more strict methods with all coupled parameters required